spec/features/card_spec.rb

```diff
@@ -33,4 +37,11 @@
       end
     end
   end
+
+  context 'When there are not card for review' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.